src/AutoCrudAdmin/Controllers/AutoCrudAdminController.cs

```diff
@@ -752,9 +753,16 @@ protected string GetComplexFormControlNameFor<T>()
             .Filterable(true)
             .Sortable(true);
 
-        if (property.PropertyType == typeof(string) && columnStringMaxLength.HasValue)
+        if (foreignKeys.Contains(property) || property.PropertyType == typeof(string))
         {
-            columnBuilder.RenderedAs((entity) => entity.ToString().ToEllipsis(columnStringMaxLength.Value));
+            if (columnStringMaxLength.HasValue)
+            {
+                columnBuilder.RenderedAs(entity => property.GetValue(entity)?.ToString().ToEllipsis(columnStringMaxLength.Value));
+            }
+            else
+            {
+                columnBuilder.RenderedAs(entity => property.GetValue(entity)?.ToString());
```

Comment:
We should have default `columnStringMaxLength`, that will be for safety. If someone didn't provide `columnStringMaxLength` and the string is too big, it will break the grid. Default should be something not too small, but big enough to still "break" the grid, but to manageble.